### DISCUSSION TITLE
Table Type & Data Reading

### DISCUSSION THREAD

**Discussion by naturallyfunny** [teamalgoritma/community#843] [Q&A] [answered]
[Data Sales-PreProcessing.xlsx](https://github.com/teamalgoritma/community/files/14078810/Data.Sales-PreProcessing.xlsx)

Yang bagian "Raw 1", bentuk table seperti itu namanya apa ya, multi-index kah? Apakah bisa diubah ke csv? Dan bagaimana cara baca data seperti itu di R ya?

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @ArdiantheGreat,

Berikut saya melampirkan file Rmd berisikan cara untuk memodifikasi file excel sehingga dapat menyerupai output yang Bapak lampirkan. 

https://drive.google.com/file/d/146T2P9dW_vC5TSOYNe_EyJTnVDBTro39/view?usp=sharing

Semoga membantu!
